cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,31 @@
 describe('', () => {
-  before(() => {
+  let articleTitle;
+  let uniqueUsername;
+  beforeEach(() => {
+    uniqueUsername = 'user' + Cypress._.random(0, 9999);
+    const uniqueEmail = uniqueUsername + '@example.com';
+    const uniquePassword = 'password' + Cypress._.random(0, 9999);
+    articleTitle = 'Test Article ' + Cypress._.random(0, 9999);
+    const articleDescription = 'This is a test article.';
 
+    cy.login(uniqueEmail, uniqueUsername, uniquePassword);
+    cy.createArticle(articleTitle, articleDescription,
+      'This is my test article');
   });
 
-  it('', () => {
-
+  it('should create an article', () => {
+    cy.visit('https://conduit.mate.academy/');
+    cy.get('a[class = "link nav-link"]')
+      .should('contain.text', 'Global Feed').click();
+    cy.contains(articleTitle, { timeout: 10000 }).should('exist');
   });
 
-  it('', () => {
-
+  it('should delete the article', () => {
+    cy.visit(`https://conduit.mate.academy/profile/${uniqueUsername}`);
```

Comment:
`https://conduit.mate.academy`
You have already used this URL 2 times in the test - to optimize the code, you can put it in a variable